apps/nwchem/readme.md

```diff
@@ -36,10 +36,44 @@ azhpc-connect -u hpcuser headnode
 ## Run the NWCHEM h2o_freq scenario
 To run on a single node with 4 cores run
 ```
-qsub -l select=1:ncpus=4:mpiprocs=4 $azhpc_dir/apps/nwchem/run_h2o_freq.sh
+qsub -l select=1:ncpus=60:mpiprocs=4 $HOME/apps/nwchem/run_h2o_freq.sh
 ```
 
 To run on two HB nodes with 8 total cores (4 cores on each node) run
 ```
 qsub -l select=2:ncpus=60:mpiprocs=4 $azhpc_dir/apps/nwchem/run_h2o_freq.sh

```

Comment:
just apps/nwchem/run_h2o_freq.sh to be consistent

Reply:
Done.